packages/common/src/spokenForms/modifiers.ts

```diff
@@ -0,0 +1,202 @@
+import {
+  ModifierType,
+  NoSpokenFormError,
+  ScopeTypeType,
+  SurroundingPairName,
+} from "..";
+
+export const modifiers: Record<ModifierType, string | null> = {
+  excludeInterior: "bounds",
+  toRawSelection: "just",
+  leading: "leading",
+  trailing: "trailing",
+  keepContentFilter: "content",
+  keepEmptyFilter: "empty",
+  inferPreviousMark: "its",
+  startOf: "start of",
+  endOf: "end of",
+  interiorOnly: "inside",
+  extendThroughStartOf: "head",
+  extendThroughEndOf: "tail",
+  everyScope: "every",
+
+  containingScope: null,
+  ordinalScope: null,
+  relativeScope: null,
+  modifyIfUntyped: null,
+  cascading: null,
+  range: null,
+};
+
+export const modifiersExtra = {
+  first: "first",
+  last: "last",
+  previous: "previous",
+  next: "next",
+  forward: "forward",
+  backward: "backward",
+};
+
+const scopes: Record<ScopeTypeType, string | null> = {
+  argumentOrParameter: "arg",
+  attribute: "attribute",
+  functionCall: "call",
+  functionCallee: "callee",
+  className: "class name",
+  class: "class",
+  comment: "comment",
+  functionName: "funk name",
+  namedFunction: "funk",
+  ifStatement: "if state",
+  instance: "instance",
+  collectionItem: "item",
+  collectionKey: "key",
+  anonymousFunction: "lambda",
+  list: "list",
+  map: "map",
+  name: "name",
+  regularExpression: "regex",
+  section: "section",
+  sectionLevelOne: "one section",
+  sectionLevelTwo: "two section",
+  sectionLevelThree: "three section",
+  sectionLevelFour: "four section",
+  sectionLevelFive: "five section",
+  sectionLevelSix: "six section",
```

Comment:
They are, but we want to be able to do spoke in form tests because we have fixtures for these. What should break? These spoken forms are only used for the Talon grammar test and nothing else.